.github/workflows/docker.yml

```diff
@@ -23,7 +25,15 @@ jobs:
             --build-arg BUILDKIT_INLINE_CACHE=1 \
             --cache-from "dependabot/dependabot-core:latest" \
             .
-      - name: Push image to packages
+      - name: Push image to packages (latest)
+        if: '!contains(github.ref, "refs/tags")'
```

Comment:
Any harm in always pushing to latest?

Reply:
No _harm_, it's just making the release process race over `:latest` by design:

Since the workflow will trigger twice for the push to main+tag creation, and those workflows will produce ~equivalent but not `===` images.
The goal was to remove ambiguity over which workflow "won", and reduce churn.

Reply:
> Since the workflow will trigger twice for the push to main+tag creation, and those workflows will produce ~equivalent but not `===` images.

Ah yes nice one, forgot about the double runs 👍 💯